testproject/tests/test_first_step_authentication.py

```diff
@@ -16,11 +16,19 @@ def test_get_emphemeral_token(active_user_with_email_otp):
     response = login(active_user_with_email_otp)
     assert response.status_code == 200
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too